tests/remote_attestation/maa/sgxlkl_enclave/oe_private/verify_cert.h

```diff
@@ -0,0 +1,12 @@
+#ifndef VERIFY_CERT_H
```

Comment:
This file should go away if we expose the cert-related APIs through OE host verify.

Reply:
Any when the hostverify lib can easily be built on Alpine...
@hukoyu This needs to be documented. It is a huge amount of code that's being duplicated from OE. Please add a note to the README file about this.